cpp_test_suite/new_tests/cxx_dserver_misc.cpp

```diff
@@ -287,6 +291,54 @@ cout << "str = " << str << endl;
 
         TS_ASSERT_THROWS_NOTHING(device1->unsubscribe_event(subscription));
     }
+
+/*
+ * Test to check that client still receives periodic archive events after
+ * polling is stopped and started again for particular device. Such scenario
+ * used to fail as described in #675.
+ */
+    void test_archive_periodic_events_after_polling_restart()
+    {
+        constexpr auto poll_period = 1000ms;
```

Comment:
std::chrono_literals::ms is C++14 standard, right? (https://en.cppreference.com/w/cpp/chrono/operator%22%22ms)
If tango-9-lts branch requires C++14 standard, we should probably upgrade the CMakeLists.txt file (https://github.com/tango-controls/cppTango/blob/tango-9-lts/CMakeLists.txt#L5) to impose at least C++14 standard, or at least for the tests part (maybe it's already there but I didn't see it).

Reply:
In https://github.com/tango-controls/cppTango/issues/626#issue-523572962 we thought about enforcing C++14 around 8/2020 in cppTango.

I would prefer having the same requirements for the tests and cppapi, as that would at least theoretically allow us to get the tests running on windows as well at some point.

Reply:
Thanks @bourtemb and @t-b for your comments. C++14 is not mandatory for this test, changing literals to explicit duration constructors is trivial. But since we agreed to move to C++14 now, let me mark your comments as resolved without changing the test.

Here is PR I opened to require C++14: https://github.com/tango-controls/cppTango/pull/732.